pkg/rule/accessrule.go

```diff
@@ -312,6 +313,22 @@ func (t Target) ToAPIDetail() types.AccessRuleTargetDetail {
 	return at
 }
 
+var slackRegex = regexp.MustCompile("@slack-(C[\\w-]+) (@\\w+)?")
```

Comment:
Not sure, shall we modify the accessrule package for this or keep it agnostic from the notification mechanism?

We can just move this to the notifiers/slack.go.

Reply:
Yeah I wasn't sure about where to put this, its easy to move and will defer to maintainers on their preference.

Reply:
I saw you changed it. I was going to say that actually other benefit of not puting it there is that if upstream decide to not merge this, we can always deploy the lambda independently, as it is self contained.